var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -340,6 +362,11 @@ def cmake_args(self):
             'DEAL_II_COMPONENT_EXAMPLES', 'examples'
         ))
 
+        # Simplex support
+        options.append(self.define_from_variant(
+            'DEAL_II_WITH_SIMPLEX_SUPPORT', 'simplex'
+        ))
+
```

Comment:
To remain organised, could we please move this to ~line 455 between "Python" and "Threading"? Sorry to be a pain about this...